aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -93,7 +91,7 @@ module AptosFramework::Account {
         writeset_epilogue_name: vector<u8>,
         currency_code_required: bool,
     ) {
-        SystemAddresses::assert_aptos_framework(account);
+        assert!(signer::address_of(account) == @aptos_framework, errors::requires_address(ENOT_APTOS_FRAMEWORK));
```

Comment:
Why not use system_addresses:: assert_aptos_framework ?

Reply:
Merge conflict resolution problem ... Resolved